IMDA-BTG/aiverify
# Dev report updates 20230606

## Description

Changes: 
- Update templates to swap the position of technical test and process checklist in the summary section.
- Update robustness plugin for textual edit
- Update shap plugin for textual edits


## Motivation and Context

[Explain the motivation or the context behind this pull request. Why is it necessary?]

## Type of Change













## How to Test

Install the updated plugins, then generate report and make sure the changes are reflected.

## Checklist

Please check all the boxes that apply to this pull request using "x":


## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]
